functions/fisher.fish

```diff
@@ -206,7 +206,7 @@ function fisher --argument-names cmd --description "A plugin manager for Fish"
                     contains -- (string lower -- $plugin) (string lower -- $commit_plugins) || set --append commit_plugins $plugin
                 end
 
-                printf "%s\n" $commit_plugins >$fish_plugins
+                printf "%s\n" $commit_plugins | string replace --regex -- '^'$HOME'/' '~/' >$fish_plugins
```

Comment:
Oops - I should not have piped from printf. I should have just made this line into:

```
string replace --regex -- '^'$HOME'/' '~/' $commit_plugins >$fish_plugins
```

If there are other changes you need, I can incorporate this one too.

Reply:
Yes, go ahead! And while you're at it, use this format for the regex to keep it consistent with the other parts of the file where we handle the ~ to $HOME conversion.

```console
string replace --regex -- $HOME \~
```

Isn't this good enough?

Reply:
> Isn't this good enough?

It is. There's an edge case where `/User/foo` includes stuff from `/User/foobar` and that string replace would make `~bar`. That is highly unlikely to be an actual issue for anyone in the real world, so I'll make the change.

Reply:
Thanks! I should look into it and update all the other instances then. Those small inconsistencies tend to add up, and after a while, one can't remember if they were intentional or not. 😂